### PR TITLE
py2 compat header

### PR DESCRIPTION
**Pull request by fmaussion**
As discussed in https://github.com/pydata/xarray/pull/1079

I just left out a couple of nearly empty ``__init__``s 

I also removed a couple of unused imports

### CODE REVIEW

**Comment by fmaussion** [comment]
The test failure seems unrelated

File: xarray/core/formatting.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+from __future__ import absolute_import
```
Comment (fmaussion) [inline xarray/core/formatting.py]: ups! yes, sorry

**Comment by shoyer** [comment]
I tried restarting the failing test which looks like a conda issue -- let's see if that helps.

**Comment by shoyer** [comment]
Thanks!

**Pull request merged**